worldedit-core/src/main/java/com/sk89q/worldedit/util/paste/EngineHubPaste.java

```diff
@@ -35,11 +34,11 @@
     private static final Pattern URL_PATTERN = Pattern.compile("https?://.+$");
 
     @Override
-    public ListenableFuture<URL> paste(String content) {
-        return Pasters.getExecutor().submit(new PasteTask(content));
+    public Callable<URL> paste(String content) {
+        return new PasteTask(content);
     }
 
-    private final class PasteTask implements Callable<URL> {
+    public static final class PasteTask implements Callable<URL> {
```

Comment:
Is there a reason this was made `public`?

Reply:
No. I remember making it static but not public :thinking: fixed